react-native-reanimated.d.ts

```diff
@@ -557,9 +557,10 @@ declare module 'react-native-reanimated' {
     finished?: boolean,
     current?: AnimatableValue
   ) => void;
+  export type EasingFactoryFunction = { factory: () => EasingFunction };
   export interface WithTimingConfig {
     duration?: number;
-    easing?: EasingFunction;
+    easing?: EasingFunction | EasingFunctionFactory;
```

Comment:
Sorry to revive a closed thread, but is this a typo? The definition is `Easing**FactoryFunction**`, but the usage is `Easing**FunctionFactory**`